### ISSUE TITLE
Cannot see (new) solutions of users i am following

### ISSUE THREAD

**Issue by Vittel** [Codewars/codewars.com#1948]
#### Describe the bug
I have recently added an intern to my follows and he added me.
He solved 3 katas by now, which i cannot see any solutions of users i am following for.

#### To Reproduce

My Account: seidchr
Other account: MoyJak

#### Expected Behavior

i can see the solutions as i can for (e.G.) user: sevensc

#### Contexts

Please complete the following information:

- Codewars username (if account related): [e.g., jhoffner]
- OS: [windows]
- Browser: [chrome]
- Browser Version: [77]

**Comment by Vittel** [comment]
just a quick update: this continues to happen on recent solutions of the user.

**Comment by kazk** [comment]
I don't think this is specific to the user. It's most likely a symptom of issues that exists for a long time (the solution fails to be grouped and doesn't show up on solutions page). I've added temporary labels to organize possibly related issues.

Can the user see their solution for "Is n divisible by x and y?"?

**Comment by kazk** [comment]
Unfortunately not. I can't think of a workaround to see specific user's solution.

If you have Codewars Red, you should be able to use Head to Head compare feature that shows the solutions side by side for all the kata you and the other user have solved.

![image](https://user-images.githubusercontent.com/639336/66807540-1db40c00-eede-11e9-94b1-137bc6d19495.png)

We'd like to eventually make it easier to review solutions to support use cases like mentoring, but there's still many other things to be done before that.

**Comment by Vittel** [comment]
Just came back to this after having a few other occurences with different collegues to brag about, just to see that you have asked a question in your first reply to this ticket :D
Sorry i didnt see that.
I will check with them when im back from vacation.

I know you are busy with this project, but could you possibly tell any timeframe on that you may possibly be able to take care of this? 1month? 5years?

**Comment by mltn123** [comment]
I'm experiencing the same issue, can't see any solutions of the user i'm following. Any advice?
E.g , solution of user "em2L" should show up for https://www.codewars.com/kata/513e08acc600c94f01000001/solutions/python/following/best_practice

**Comment by hobovsky** [comment]
@mltn123 Some solutions of the user you mentioned are not visible because of a bug, which makes solutions not appear on solutions board. Unfortunately, the bug manifests itself quite often and other than resubmitting a slightly modified solution (added or removed comment or renamed variable), or manual intervention from a moderator, it's difficult to fix. I was able to fix one of their solutions, I will try to fix others too.

One thing you have to make sure though is that the user clicks "Submit final" button when his tests pass. On two or three kata, they have not clicked it, and they'd have to resubmit.

**Comment by Blind4Basics** [comment]
I checked the user and his solution is supposed to be visible (finalized and grouped). I believe it's actually related to this: https://github.com/codewars/codewars.com/issues/2263

**Comment by mltn123** [comment]
I see, I appreciate the quick response and help.
Thanks
